lib/import-export-visitor.js

```diff
@@ -114,9 +125,14 @@ module.exports = class ImportExportVisitor extends Visitor {
     const directives = parent.directives;
     if (directives) {
       const directiveCount = directives.length;
-
       for (let i = 0; i < directiveCount; ++i) {
-        insertCharIndex = Math.max(directives[i].end, insertCharIndex);
+        const d = directives[i];
```

Comment:
If Babylon represents directives with `.directives` does Acorn do something different? Is there a potential gap?